lib/ui/compositing.dart

```diff
@@ -133,10 +133,10 @@ class SceneBuilder extends NativeFieldWrapperClass2 {
   /// Rasterization will be clipped to the given shape.
   ///
   /// See [pop] for details about the operation stack.
-  void pushPhysicalShape({ Path path, double elevation, Color color }) {
-    _pushPhysicalShape(path, elevation, color.value);
+  void pushPhysicalShape({ Path path, double elevation, Color color, Color shadowColor}) {
+    _pushPhysicalShape(path, elevation, color.value, shadowColor?.value ?? 0);
```

Comment:
is 0 the right default?

Reply:
Ahh no, should be SK_ColorBLACK https://github.com/google/skia/blob/master/include/core/SkColor.h#L68 - Fixed